<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
    background-color: silver;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="2" column="1">
       <widget class="QPushButton" name="push_2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="push_6">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="push_5">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="push_3">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="push_4">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="push_7">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="push_divided">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="push_8">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="push_squared">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="push_1">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="push_plus">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="push_times">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="push_0">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="push_minus">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="push_9">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="push_clear">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="push_squareroot">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="push_equals">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: grey;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="push_negative">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>-/+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="push_point">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px white;
}
QPushButton:pressed {
    background-color: cornflowerblue;
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
